.github/workflows/python-app.yml

```diff
@@ -46,7 +46,7 @@
        working-directory: ./tests
        run: |
          source ../.venv/bin/activate       
-         pytest
+         pytest --cov --cov-fail-under=45
```

Comment:
In my local testing, we had a total of ~46%, so as a first make sure we don't do *worse* when adding new code.  As we go along, we should be able to bump this up higher as well.